preprocessing/constants.py

```diff
@@ -0,0 +1,175 @@
+# %% Name of different type of form
+formtype_names = ["household", "business", "service", "large_scale_farm", "local_aut"]
+
+MONTHS_PREFIX = "G_1b"
+
+# %% defaults usage time windows in a day
+USAGE_WD_DEFAULTS = {
+    "0-7": None,
+    "7-10": None,
+    "10-12": None,
+    "12-18": None,
+    "18-22": None,
+    "22-24": None,
+}
+
+# %% defaults months and day numaration
+
+months_defaults = {
+    "January": 1,
+    "February": 2,
+    "March": 3,
+    "April": 4,
+    "May": 5,
+    "June": 6,
+    "July": 7,
+    "August": 8,
+    "September": 9,
+    "October": 10,
+    "November": 11,
+    "December": 12,
+}
+
+working_day = {
+    "monday": 0,
+    "tuesday": 1,
+    "wednesday": 2,
+    "thursday": 3,
+    "friday": 4,
+    "saturday": 5,
+    "sunday": 6,
+}
+
+# %% Defaults constants used in preprocessing
+
+exc_rate = 0.05334  # $/ZAR exchange rate
+large_buck = 10
+medium_buck = 5
+small_buck = 1
+buck = 5
+
+DENSITY_DICT = {
+    "g/biogas_density": 0.0012,
+    "h/biofuel_density": 0.9,
+    "i/kerosene_density": 0.8,
+    "j/LPG_density": 0.55,
+    "k/eth_alc_density": 0.789,
```

Comment:
What is the source of those data?

Reply:
@Bachibouzouk those data have been put by Simone. I asked him where they come from and he said that he have to look for the sources because he did that research quite some time ago. He will come back with the info soon.